src/coreclr/vm/stubmgr.cpp

```diff
@@ -938,10 +938,15 @@ BOOL ThePreStubManager::DoTraceStub(PCODE stubStartAddress, TraceDestination *tr
     // We cannot tell where the stub will end up
     // until after the prestub worker has been run.
     //
+    #if defined(TARGET_ARM64) && defined(__APPLE__)
```

Comment:
We never indent the preprocessor directives themselves in coreclr.

Basically, every line starting with an `#` should start at column 0 on the left. The code it is protected should be indented as done here.